#
# Ensure that the root file system is remounted with the correct
# options from fstab
#
--- old/checkroot.sh	2005-05-28 19:30:55.488975598 -0700
+++ new/checkroot.sh	2005-05-28 19:31:32.334390450 -0700
@@ -149,7 +149,7 @@
 #	is on a ro fs until the remount succeeded. Then clean up old mtabs
 #	and finally write the new mtab.
 #
-mount -n -o remount,$rootmode /
+mount -n -o remount,$rootopts /
 if test "$rootmode" = rw
 then
 	if test ! -L /etc/mtab
